src/sources/exec/mod.rs

```diff
@@ -773,8 +773,11 @@ mod tests {
             run_command(config.clone(), hostname, decoder, shutdown, tx),
         );
 
-        let timeout_result =
-            crate::test_util::components::assert_source_compliance(&[], timeout).await;
+        let timeout_result = crate::test_util::components::assert_source_compliance(
+            &crate::test_util::components::SOURCE_TAGS,
```

Comment:
Presumably this is because the test is `#[cfg(unix)]`, so it is using the fullpath to avoid having to also `#[cfg(unix)]` the import.

Arguably a lazy move..